console/src/main/webapp/WEB-INF/views/passwordRecoveryForm.jsp

```diff
@@ -90,7 +94,12 @@
 	<script type="text/javascript">
     /* Validate the form */
     function validate() {
+<c:when test="${recaptchaActivated}">
```

Comment:
needs to be wrapped within `<c:choose>` & </c:choose>`

Reply:
Wooops sorry ...